mvpa2/clfs/transerror.py

```diff
@@ -984,10 +988,13 @@ def plot(self, labels=None, numbers=False, origin='upper',
         NlabelsNN = len(non_empty)
         Nlabels = len(labels_plot)
 
-        if matrix.shape != (NlabelsNN, NlabelsNN):
-            raise ValueError, \
-                  "Number of labels %d doesn't correspond the size" + \
-                  " of a confusion matrix %s" % (NlabelsNN, matrix.shape)
+        # I believe this Error is superfluous -- or rather: wouldn't let
+        # subsets of matrices be printed.
```

Comment:
do we want subsets of matrices printed?  that is related to the comment above.  It could be a feature indeed, but I think if desired, it should be explicit, like adding an additional parameter `allow_partial=False`, so by default we would blow (so not to miss errors in specification) and only if explicitly instructed -- then to be allowed.